src/languages/scss.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  argumentMatcher,
+  cascadingMatcher,
+  conditionMatcher,
+  createPatternMatchers,
+  matcher,
+  patternMatcher,
+} from "../util/nodeMatchers";
+import {
+  NodeMatcherAlternative,
+  ScopeType,
+  SelectionWithEditor,
+} from "../typings/Types";
+import { patternFinder } from "../util/nodeFinders";
+import { getNodeRange, selectChildrenWithExceptions } from "../util/nodeSelectors";
+import { SyntaxNode } from "web-tree-sitter";
+
+const STATEMENT_TYPES = [
+  "apply_statement",
+  "at_rule",
+  "charset_statement",
+  "debug_statement",
+  "declaration",
+  "each_statement",
+  "error_statement",
+  "for_statement",
+  "forward_statement",
+  "function_statement",
+  "if_statement",
+  "import_statement",
+  "include_statement",
+  "keyframes_statement",
+  "media_statement",
+  "mixin_statement",
+  "namespace_statement",
+  "placeholder",
+  "rule_set",
+  "supports_statement",
+  "use_statement",
+  "warn_statement",
+  "while_statement",
+];
+
+const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
+  // map, list not supported in tree-sitter version
+  ifStatement: "if_statement",
+  condition: conditionMatcher("condition"),
+  // Currently data-lang=^"tel" not treated as a statement
```

Comment:
Happy to drop this but wanted to call it out.

Reply:
Is this something we should file as a bug?